rr_iarrc/launch/cone_bottom_detector.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-    <node name="cone_bottom_detector" pkg="rr_iarrc" type="cone_bottom_detector" output="screen">
+<node name="cone_bottom_detector" pkg="rr_iarrc" type="cone_bottom_detector" output="screen">
```

Comment:
Might want to note that this node is meant to be used along with the pcl overhead converter node.